.github/workflows/turborepo-test.yml

```diff
@@ -382,8 +382,10 @@ jobs:
         run: |
           cargo groups test turborepo-libraries
 
-  cleanup:
-    name: Cleanup
+  summary:
+    name: Turborepo Test Summary
```

Comment:
This needs to be added to the branch protection rules for main https://github.com/vercel/turbo/settings/branch_protection_rules/23353343